jsx_test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Card from "./card";
+
+export default class CardList extends React.Component {
+  constructor(props)
```

Comment:
Opening curly brace does not appear on the same line as controlling statement  brace-style